net/croc/Makefile

```
@@ -0,0 +1,51 @@
+# SPDX-License-Identifier: GPL-2.0-only
+#
+# Copyright (C) 2023 Jonas Jelonek
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=croc
+PKG_VERSION:=9.6.5
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/schollz/croc.git
+PKG_MIRROR_HASH:=324ab1ef98b584904391e55b4f37a1a24fb089303c49a27eeeb4881e7d2d2e82
+PKG_SOURCE_VERSION:=v$(PKG_VERSION)
+
+PKG_LICENSE:=MIT
+PKG_MAINTAINER:=Jonas Jelonek <jelonek.jonas@gmail.com>
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/croc-$(PKG_VERSION)
+PKG_BUILD_DEPENDS:=golang/host
+PKG_BUILD_PARALLEL:=1
+PKG_BUILD_FLAGS:=no-mips16
+
+GO_PKG:=github.com/schollz/croc
```

```suggestion
GO_PKG:=github.com/schollz/croc/v9
GO_PKG_BUILD_PKG:=$(GO_PKG)
```

`go.mod` advertises it's `github.com/schollz/croc/v9`
set `GO_PKG_BUILD_PKG` to build main package (croc) only

---

done